File: client/src/app/_models/hr/intervw.ts
import { IIntervwItem } from "./intervwItem";

export interface IIntervw{
     id: number;
     orderId: number;
     orderNo: number;
     orderDate: Date;
     customerId: number;
     customerName: string;
     interviewMode: string;
     interviewDateFrom: Date;
     interviewDateUpto: Date;
     interviewStatus: string;
     interviewVenues: string;
     interviewItems: IIntervwItem[];
}

File: api/Entities/Admin/Intervw.cs
namespace api.Entities.Admin
{
    public class Intervw: BaseEntity
    {
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public DateTime InterviewDateFrom { get; set; }
        public DateTime InterviewDateUpto { get; set; }
        public string InterviewVenues { get; set; }
        public string InterviewStatus { get; set; }
        public ICollection<IntervwItem> InterviewItems { get; set; }
    }
}